src/main/res/values/strings.xml

```diff
@@ -140,17 +140,17 @@
   <string name="unknown_format">Unknown: %s</string>
   <string name="unsigned">Unsigned</string>
   <string name="unsigned_description">Unsigned. Could not verify the application list. Be careful downloading
-    application from unsigned repositories.</string>
+    applications from unsigned repositories.</string>
   <string name="unstable_updates">Unstable updates</string>
-  <string name="unstable_updates_summary">Suggest upgrades to unstable versions</string>
+  <string name="unstable_updates_summary">Suggest installing unstable versions</string>
   <string name="unverified">Unverified</string>
   <string name="update">Update</string>
-  <string name="update_notifications">Notifications about new applications</string>
+  <string name="update_notifications">Notifications about newer applications</string>
```

Comment:
This line is single line at the application preferences screen. There is a switch button right to the text, so "Notifications about newer applications" will be truncated, probably. Let's keep short "Update notifications".

Reply:
Also, I already said somewhere that the client doesn't notify about _new applications_ (which can mean, for examples, "new applications added to repository"). It notifies about _new version of already installed applications_.